BUILDER_DEV.md

```diff
@@ -1,390 +1,275 @@
+$ sudo apt-get install npm
+```
+
+* Cd into the builder-web component
+
+```
+$ cd path/to/your/habitat/repo/components/builder-web
+```
+
+* Install all dependencies
+
 ```
+$ npm install
+```
+
+* Create the habitat.conf.js file
+
+```
+$ cp habitat.conf.sample.js habitat.conf.js
+```
+
+* Start the UI
+
+```
+$ sudo npm start
+```
+
+* Open up a browser and navigate to http://localhost:3000/#/pkgs - you should see the Habitat UI running locally.
 
-## Build the Builder services for the first time
-1. Open a new terminal window.
-1. Run `make build-srv`
+### Creating an Origin
 
-Note: this only needs to be done the first time to get the pieces in place. If you clean the project, make sure to unset the variables show below when you do a clean build, otherwise the build will look for packages on your local build service!
+* Navigate to http://localhost:3000/#/pkgs
+* Log in
+* Click on "Create New Origin"
+* Fill out form (call the origin "core"), click "Save & Continue"
 
-## Run the Builder services
-1. Open a new terminal window.
-1. Export the following environment variables:
+### Install Dependencies in your local Builder Env
+
+* on your VM, go to the production instance of Builder (the one at habitat.sh)
+* Click on "My origins"
+* Click on "core"
+* Download public and private keys for the "core" origin (make sure the timestamp on the public key matches the one on the private key!) - we currently need the keys for the core production origin in order to upload some dependencies to our local Builder env)
+* From a terminal, run
 
 ```
-export HAB_AUTH_TOKEN=<your github token>
-export HAB_BLDR_URL=http://localhost:9636
-export HAB_ORIGIN=<your origin>
+$ hab origin key import
 ```
 
-Now run `sudo -E make bldr-run` from the root of your hab repo.
+* Paste the production core origin public key contents
+* Hit Ctrl + D twice
+* Run this again:
 
-The first time this command runs, it will create the required databases. Let it run for a while, and then re-start it if there are errors (this is normal for the first time setup).
+```
+$ hab origin key import
+```
 
-When `make bldr-run` can proceed with all the services up and running, you are ready to proceed to the next step.
+* Paste the production core origin private key contents
+* Hit Ctrl + D twice
+* Run
+```
+$ hab install core/hab-backline
+$ cd /hab/cache/artifacts
+$ hab pkg upload -c stable core-hab-backline...hart
+```
+* You will need to follow this same process for any dependencies of anything you want to build locally - the sample app I will use below depends on [core/scaffolding-node](https://bldr.habitat.sh/#/pkgs/core/scaffolding-node/latest), [core/git](https://bldr.habitat.sh/#/pkgs/core/git/latest), [core/node](https://bldr.habitat.sh/#/pkgs/core/node/latest), [core/busybox-static](https://bldr.habitat.sh/#/pkgs/core/busybox-static/latest) - so I will follow the same procedure to download them from production builder, then upload them to my local builder.
```

Comment:
Minor - may want to switch back out of first person to be consistent with the rest of the doc.